### ISSUE TITLE
easy_install jsonrpclib fails with error: [Errno 2] No such file or directory: 'README.md'

### ISSUE THREAD

**Issue by mksteele** [joshmarshall/jsonrpclib#41]
Looks like something went wrong after the version bump:

miranda@ubuntu:~/ir-miranda$ sudo easy_install jsonrpclib
Searching for jsonrpclib
Reading https://pypi.python.org/simple/jsonrpclib/
Best match: jsonrpclib 0.1.4
Downloading https://pypi.python.org/packages/source/j/jsonrpclib/jsonrpclib-0.1.4.tar.gz#md5=a80e95fd1f9437736e2ae894a877d7c8
Processing jsonrpclib-0.1.4.tar.gz
Writing /tmp/easy_install-OCR0tL/jsonrpclib-0.1.4/setup.cfg
Running jsonrpclib-0.1.4/setup.py -q bdist_egg --dist-dir /tmp/easy_install-OCR0tL/jsonrpclib-0.1.4/egg-dist-tmp-a6jcXZ
error: [Errno 2] No such file or directory: 'README.md'

**Comment by cloudnull** [comment]
+1 this is an issue for us too. We're no longer able to build or install jsonrpclib using pip.

``` shell
root@server1:~# pip wheel jsonrpclib==0.1.4
You are using pip version 7.1.0, however version 7.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting jsonrpclib==0.1.4
  Using cached jsonrpclib-0.1.4.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-jUOSl3/jsonrpclib/setup.py", line 28, in <module>
        long_description = open("README.md").read()
    IOError: [Errno 2] No such file or directory: 'README.md'
``` 

This was noticed today during our gate jobs as found here: http://logs.openstack.org/14/231214/5/check/gate-openstack-ansible-dsvm-commit/dcf2d00/console.html#_2015-10-08_21_41_55_601

**Comment by the2hill** [comment]
We are also experiencing the same errors with the latest version and cloudnull's trace is exactly what we are seeing.
I believe the fix is to include README.md in the Manifest. 

'''
include README.md
'''

**Comment by mriedem** [comment]
openstack CI systems are also hitting this:

https://bugs.launchpad.net/openstack-gate/+bug/1504372

**Comment by joshmarshall** [comment]
Thanks folks for bringing this to my attention -- I apologize for the error and any workarounds it caused. I attempted to hotfix with a MANIFEST.in change, adding README.md, but that didn't fix the issue, so instead I've made the long description "optional" if the file exists. I've pushed 0.1.6 to PyPI, hopefully build systems and other environments will succeed now, let me know otherwise.

Thanks (and sorry) again.
